src/components/networks/createNetworkDialog.jsx

```diff
@@ -386,7 +390,7 @@ class CreateNetworkModal extends React.Component {
                                        onValueChanged={this.onValueChanged} />
                 { (this.state.forwardMode === "nat" || this.state.forwardMode === "route") &&
                 <NetworkDeviceRow dialogValues={this.state}
-                                  devices={this.props.devices}
+                                  devices={getNetworkDevices()}
```

Comment:
Anoter nitpick, but since we are now trying to avoid unnecessary passing of arguments, maybe we can call `getNetworkDevices()` directly in `NetworkDeviceRow` component, and get rid of "devices" argument.

Reply:
Yes, good idea.